Stamen_Maps_for_Spatial_Data.Rmd

```diff
@@ -0,0 +1,135 @@
+# Using Stamen Maps for Plotting Spatial Data
+
+Kumari Nishu and Neelam Patodia
+
+Objective: 
+We intend to highlight the usability of Stamen Maps for visualizing spatial data. 
+
+Approach:
+We conduct a comparative study between different types of graphs to understand the visualization tool which best demonstrates spatial data.We use the publicly available data on the number of vehicle collisions in New York for this demonstration.   
+
+
+```{r setup, include=FALSE}
+ # keep this chunk in your .Rmd file
+ knitr::opts_chunk$set(warning = FALSE, message = FALSE)
+```
+
+```{r}
+library(caret)
+library(dplyr)
+library(ggmap)
+library(ggplot2)
+library(forwards)
+#library(plyr)
+library(gridExtra)
+
+#Loading the collisions dataset available from "https://data.cityofnewyork.us/Public-Safety/NYPD-Motor-Vehicle-Collisions-Crashes/h9gi-nx95/data"
+raw_df<-read.csv(file="/Users/neelampatodia/Downloads/collisions_data.csv", header=TRUE, sep=",")
```

Comment:
Please add `collisions_data.csv` to the resources folder and then change this link to that location.

Reply:
A sample of the collisions data has been added to the resources and the link updated.